CHANGELOG.md

```diff
@@ -1,13 +1,10 @@
 # AWS Mobile SDK for iOS CHANGELOG
-## Unreleased Changes
-- Added support for adding custom demographic to pinpoint [PR: #2410](https://github.com/aws-amplify/aws-sdk-ios/pull/2410)
 
-## Unreleased
+## 2.13.2
 
 ### New features
 - **AWSMobileClient** Added IdentityProvider strings for Sign in with Apple: `IdentityProvider.apple`. See [Issue #1809](https://github.com/aws-amplify/aws-sdk-ios/issues/1809)) and [PR #2425](https://github.com/aws-amplify/aws-sdk-ios/pull/2425).
```

Comment:
Looks like our previous pr has subsection for each service/type. So maybe change to this:

```
### New features
- **AWSMobileClient**
  - Added IdentityProvider strings for Sign in with Apple: `IdentityProvider.apple`. See [Issue #1809](https://github.com/aws-amplify/aws-sdk-ios/issues/1809)) and [PR #2425](https://github.com/aws-amplify/aws-sdk-ios/pull/2425).

- AWS Pinpoint
 - Added support for adding custom demographic to pinpoint [PR: #2410](https://github.com/aws-amplify/aws-sdk-ios/pull/2410)
```

Reply:
Should we move the line `Added support for Pinpoint Journey Push Notifications` to New Features?

Reply:
> Should we move the line `Added support for Pinpoint Journey Push Notifications` to New Features?

good idea, let me find the PR as well